example/.gitignore

```diff
@@ -1 +1,3 @@
-.env
\ No newline at end of file
+.env
+.venv_*
```

Comment:
what is .venv and .pyc?

Reply:
.venv_* is the folder hosting the python virtual environment and *.pyc are the cached python files